### ISSUE TITLE
Check for typo and language mistakes

### ISSUE THREAD

**Issue by Antondomashnev** [dblock/danger-changelog#23]
Sometimes typos or incorrect spelling appear in the changelog, although it can be prevented by danger.
I think as a reference we can take a look on [this](https://github.com/dbgrandi/danger-prose).

**Comment by dblock** [comment]
Did you try to use danger-prose along with this one? I think it might just work out of the box?

**Comment by Antondomashnev** [comment]
@dblock not really, didn't have time yet. The idea just came up and I decided to write it down before I forget 😄

**Comment by dblock** [comment]
@Antondomashnev Want to help out with this gem? Make a release? I would be happy to add you to contributors and rubygems! (what's your rubygems email?)

**Comment by Antondomashnev** [comment]
@dblock I'm already 😉

**Comment by dblock** [comment]
Ah yes, I forgot :)
